etl/renv.lock

```diff
@@ -2808,6 +2808,19 @@
       ],
       "Hash": "ad57b543f7c3fca05213ba78ff63df9b"
     },
+    "sfarrow": {
```

Comment:
This was never added to renv despite being used in the airport noise raster script.